lib/src/darwin.dart

```diff
@@ -246,10 +273,10 @@ class AVAudioSession {
 
   void close() {
     _interruptionNotificationSubject.close();
-    //_routeChangeSubject.close();
-    //_silenceSecondaryAudioHintSubject.close();
-    //_mediaServicesWereLostSubject.close();
-    //_mediaServicesWereResetSubject.close();
+    _routeChangeSubject.close();
+    _silenceSecondaryAudioHintSubject.close();
+    _mediaServicesWereLostSubject.close();
+    _mediaServicesWereResetSubject.close();
```

Comment:
I just realised it probably doesn't make sense to have a `close` method in this class anyway. It's a singleton, and once it's closed it effectively becomes unusable. `close` doesn't release any platform resources, and these stream controllers will all disappear when the FlutterEngine is destroyed.

Reply:
Removed the close function